### ISSUE TITLE
Feature request; a default dummy gun progression

### ISSUE THREAD

**Issue by DigitalDuelist** [gisellevonbingen-Minecraft/MineColonies_Compatibility#19]
From what I've seen about how this mod programs guns it's not possible for all gun mods to be automatically supported. However, it's also hard to find guns that have an actual progression that fits inside minecolonies's own, which in the short term requires players regularly double check that their guard towers don't get gunners when they can't possibly equip those gunners

My suggestion for both is to make a basic gun at wood-stone-gold-iron-diamond-netherite tiers and just have a quick simple and easy implementation, that way you always have the option of having a gunner if you want even if you only have a mod with a single gun in it. It would also let you have a config with an array you can fill for each gun tier and toss whatever items in there you want and the gunner will pretend it's actually the default weapon.

Cons; Adds an entire gun system. Even if you cut every corner you can, it's still a non-trivial amount of work
Pros; A default so you can support more guns from more mods without actually needing to hard-code them all, as well as resolving the issue of getting a gunner automatically put in your guard tower without having the ability to actually equip them

**Comment by gisellevonbingen** [comment]
This is actually a good suggestion.
But, the Gunner is a job that was added from the beginning to be compatible with guns from other modes only.
It's better to hire as an archer if there no guns.

**Comment by gisellevonbingen** [comment]
Or, if there is a mod similar to your suggestion, please let me know. Let's review it.

**Comment by DigitalDuelist** [comment]
https://www.curseforge.com/minecraft/mc-mods/old-guns-mod
https://www.curseforge.com/minecraft/mc-mods/ydms-gunblades-mod
https://www.curseforge.com/minecraft/mc-mods/yamato-gun

I was a bit surprised at how few gun mods there were like I had implied, but I did at least find a few options. The above options are the best fits I could find without using the search function, and the below is a perfect fit... sitting at 560~ downloads lol

https://www.curseforge.com/minecraft/mc-mods/simple-vanilla-guns 

As far as the gunner requiring guns from other mods; I did recognize that was the intention, my issue was more with the fact that I can have a gunner automatically hired by the colony in spite of not having a gun mod. Which is more of an issue while I've left the server running or am playing with a different part of the modpack and my knight or archer gets killed and so a gunner gets put in it's place while I'm not there to undo it. I have had 5 different occasions where I've returned with 3 or more of my guards turned into useless gunners while I'm working on my modpack. Hence why I looked at how I would add compat with a random dummy item for now, noticed I couldn't, and suggested a way that would decrease the workload, even though I honestly have no idea lol
